### PR TITLE
Add hw4 ex1

### PR DESCRIPTION
**Pull request by eogenio777**

### CODE REVIEW

File: .allure/allure-2.8.1/bin/allure [hunk 1]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (akulebyakin) [inline .allure/allure-2.8.1/bin/allure]: Please add `.allure` folder to `.gitignore` and remove it from remote repository

File: src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        HomePage homePage = new HomePage(driver);
+
+        step("Assert Browser title");
+        softAssertions.assertThat(homePage.getTitle()).isEqualTo(title);
+
+        step("Perform login");
+        homePage.topMenu().clickLoginForm();
+        homePage.topMenu().sendKeysName(properties.getProperty("username"));
+        homePage.topMenu().sendKeysPassword(properties.getProperty("password"));
+        homePage.topMenu().clickLoginButton();
```
Comment (akulebyakin) [inline src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java]: What do you think about moving login steps to HomePage or TopMenu page object?
Login can be used in many tests and usually it is a part of PO

File: src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+import static io.qameta.allure.Allure.step;
+
+import com.epam.tc.hw3.page.objects.composite.HomePage;
+import com.epam.tc.hw3.page.objects.composite.component.FrameComponent;
+import com.epam.tc.hw4.BaseExerciseTestHW4;
+import com.epam.tc.hw4.storynames.Tags;
+import com.epam.tc.hw4.utils.DataProviderForHW4;
+import io.qameta.allure.Description;
+import io.qameta.allure.Epic;
```
Comment (akulebyakin) [inline src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java]: unused import can be removed
```java
import io.qameta.allure.Epic;
```

File: src/main/java/com/epam/tc/hw4/utils/AttachmentUtils.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw4.utils;
+
+import io.qameta.allure.Allure;
+import io.qameta.allure.Attachment;
+import java.util.List;
+import lombok.experimental.UtilityClass;
```
Comment (akulebyakin) [inline src/main/java/com/epam/tc/hw4/utils/AttachmentUtils.java]: Unused imports can be removed
```java
import io.qameta.allure.Allure;
import java.util.List;
```

File: src/test/java/com/epam/tc/hw4/BaseExerciseTestHW4.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import io.qameta.allure.Description;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.ITestContext;
+import org.testng.ITestResult;
```
Comment (akulebyakin) [inline src/test/java/com/epam/tc/hw4/BaseExerciseTestHW4.java]: Unused import can be removed
```java
import org.testng.ITestResult;
```

File: src/test/java/com/epam/tc/hw4/utils/DataProviderForHW4.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
+package com.epam.tc.hw4.utils;
+
+import java.util.Arrays;
+import java.util.List;
+import lombok.experimental.UtilityClass;
```
Comment (akulebyakin) [inline src/test/java/com/epam/tc/hw4/utils/DataProviderForHW4.java]: Unused import can be removed
```java
import lombok.experimental.UtilityClass;
```

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Please review comments

File: .allure/allure-2.8.1/bin/allure (hunk 1 above)
Comment (eogenio777) [reply .allure/allure-2.8.1/bin/allure]: Removed from repository and .gitignore is refactored.

File: src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java (hunk 2 above)
Comment (eogenio777) [reply src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java]: Created a method performLogin in TopMenu class.

File: src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java (hunk 3 above)
Comment (eogenio777) [reply src/test/java/com/epam/tc/hw4/ex1/Exercise1Test.java]: Import removed

File: src/main/java/com/epam/tc/hw4/utils/AttachmentUtils.java (hunk 4 above)
Comment (eogenio777) [reply src/main/java/com/epam/tc/hw4/utils/AttachmentUtils.java:4]: Import removed

File: src/test/java/com/epam/tc/hw4/BaseExerciseTestHW4.java (hunk 5 above)
Comment (eogenio777) [reply src/test/java/com/epam/tc/hw4/BaseExerciseTestHW4.java]: Import removed

File: src/test/java/com/epam/tc/hw4/utils/DataProviderForHW4.java (hunk 6 above)
Comment (eogenio777) [reply src/test/java/com/epam/tc/hw4/utils/DataProviderForHW4.java]: Import removed

**Pull request merged**